catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/ops/IcebergTableOpsHelper.java

```diff
@@ -156,7 +160,9 @@ private void doUpdateColumnType(
     icebergUpdateSchema.updateColumn(fieldName, (PrimitiveType) type);
   }
 
-  private ColumnPosition getAddColumnPosition(StructType parent, ColumnPosition columnPosition) {
+  // Iceberg doesn't support LAST
```

Comment:
Iceberg doesn't support LAST, so what do you do in the method?

Reply:
this method transform `Last` Position to `After`, or `first`

Reply:
update the comment